examples/js/loaders/GLTFLoader.js

```diff
@@ -1299,7 +1299,7 @@ THREE.GLTFLoader = ( function () {
 
 			if ( typeof gltfDef.extras === 'object' ) {
 
-				object.userData = gltfDef.extras;
+				object.userData = Object.assign(object.userData, gltfDef.extras);
```

Comment:
Object.assign modifies its first argument, and threejs' styleguide uses spaces inside parens. So instead:

```js
Object.assign( object.userData, gltfDef.extras );
```